### DISCUSSION TITLE
Access to r14 etc. PvP Gear

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#3457] [Suggestions]
One possibility, although likely more complex to implement, is gating access to r14 and the set by a combination of weeks and honored kills (potentially per account or per char).

numbers are adjustable ofc:
base price per item 50-60k

week 1: only access for people with 75k or more hk
week 2: only access for people with 50k or more hk
week 3: only access for people with 25k or more hk
week 4: only access for people with 10k or more hk
week 5: only access for people with 2,5k or more hk
week 6: everyone gets access

**Comment by Vitusin** [comment]
Just add it to game and make cost 75k for each part. Problem solved.But do already something.!
